addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketProxy.java

```diff
@@ -891,11 +887,8 @@ public void removeSenderListener(WebSocketSenderListener senderListener) {
     private static synchronized Comparator<WebSocketSenderListener> getSenderListenersComparator() {
         if (null == senderListenersComparator) {
             senderListenersComparator =
-                    new Comparator<WebSocketSenderListener>() {
-                        @Override
-                        public int compare(WebSocketSenderListener o1, WebSocketSenderListener o2) {
-                            return Integer.compare(o1.getListenerOrder(), o2.getListenerOrder());
-                        }
+                    (WebSocketSenderListener o1, WebSocketSenderListener o2) -> {
```

Comment:
This one was not tweaked (the types can be removed as well, same in other places).